pkg/staticnetworkconfig/generator_test.go

```diff
@@ -424,6 +544,74 @@ var _ = Describe("StaticNetworkConfig", func() {
 	})
 })
 
+var _ = Describe("StaticNetworkConfig.GenerateStaticNetworkConfigDataYAML - generate nmpolicy", func() {
+	var (
+		staticNetworkGenerator = snc.New(logrus.New())
+		hostsYAML              = `[
+	{
+		"network_yaml": "%s"
+	}
+]`
+		hostsYAMLAndIni = `[
+	{
+		"network_yaml": "%s",
+        "mac_interface_map": %s
+	}
+]`
```

Comment:
For values that are so short I'd rather see this in one line than break up the indentation like this.